esp_wrover_kit/include/bsp/display.h

```diff
@@ -55,13 +63,23 @@ extern "C" {
  * spi_bus_free(spi_num_from_configuration);
  * \endcode
  *
+ * @param[in]  config    display configuration
  * @param[out] ret_panel esp_lcd panel handle
  * @param[out] ret_io    esp_lcd IO handle
  * @return
- *      - ESP_OK                On success
- *      - Else                  esp_lcd failure
+ *      - ESP_OK         On success
+ *      - Else           esp_lcd failure
+ */
+esp_err_t bsp_display_new(bsp_display_config_t *config, esp_lcd_panel_handle_t *ret_panel, esp_lcd_panel_io_handle_t *ret_io);
+
+/**
+ * @brief BSP display default configuration
+ *
  */
-esp_err_t bsp_display_new(esp_lcd_panel_handle_t *ret_panel, esp_lcd_panel_io_handle_t *ret_io);
+#define BSP_DISPLAY_CONFIG()           \
+    {                                       \
+        .max_transfer_sz = BSP_LCD_DRAW_BUFF_SIZE*sizeof(uint16_t), \
```

Comment:
This definition relies on `BSP_LCD_DRAW_BUFF_SIZE`, which is defined in different header file: `esp_wrover_kit.h`

Reply:
Right, I included header `esp-bsp.h`.